src/main/resources/couchbase_view/all.js

```diff
@@ -0,0 +1,5 @@
+function (doc, meta) {
```

Comment:
Missing name in function declaration.<br>'function' is defined but never used.<br>'meta' is defined but never used.